.github/workflows/lib-lint-and-test.yaml

```
@@ -1,12 +1,13 @@
 name: PR Checks
```

Let's rename it to align with other workflows names:
```suggestion
name: Geti Tune Lib checks
```